src/client/MenuWindow.cpp

```diff
@@ -178,29 +178,26 @@ bool MenuWindow::clickAction(sf::Vector2i clickPoint, int index, bool isOnButton
 {
     currentMenu->at(index).setInactive();
 
-    if (gameEnginePtr->intersectPointRect(  
-        clickPoint, 
-        currentMenu->at(index).buttonRect->getGlobalBounds()))
+    if (gameEnginePtr->intersectPointRect(
+            clickPoint,
+            currentMenu->at(index).buttonRect->getGlobalBounds()))
     {
         isOnButton = currentMenu->at(index).clickButton();
     }
-    if (isOnButton && index==BUTTON_CREAT && currentMenu == &menuButtons)
+    if (isOnButton && index == BUTTON_CREAT && currentMenu == &menuButtons)
     {
         currentMenu = &newGameButtons;
         currentText = &newGameTexts;
     }
-    else if (isOnButton && index==BUTTON_CONNECT 
-                        && currentMenu == &menuButtons)
+    else if (isOnButton && index == BUTTON_CONNECT && currentMenu == &menuButtons)
     {
         return connectToGame(menuButtons[BUTTON_ID]());
     }
-    else if (isOnButton && index==BUTTON_LOAD 
-                        && currentMenu == &newGameButtons)
+    else if (isOnButton && index == BUTTON_LOAD && currentMenu == &newGameButtons)
     {
         std::cout << "Load a Game: Not implemented today\n";
     }
-    else if (isOnButton && index==BUTTON_START 
-                        && currentMenu == &newGameButtons)
+    else if (isOnButton && index == BUTTON_START && currentMenu == &newGameButtons)
```

Comment:
Same

Reply:
For these one I would rather prefer to keep it on a single line. Indeed it's under a hundred char and if we want to do a new line it would not make much sense to do it for 2 conditions and not three. But three lines here would be to much for that I think